src/System.Runtime.InteropServices.RuntimeInformation/src/System.Runtime.InteropServices.RuntimeInformation.builds

```diff
@@ -15,13 +15,22 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Runtime.InteropServices.RuntimeInformation.csproj">
-      <TargetGroup>net46</TargetGroup>
+      <OSGroup>Windows_NT</OSGroup>
```

Comment:
This doesn't make sense to me.  I thought these target groups (net45, netcore50, netcore50aot) were only included in the win runtime package.